.github/workflows/infrastructure.yml

```diff
@@ -23,150 +23,418 @@
+            --query peeringState)"
+
+      - name: Container app environment 🗃️
+        uses: Azure/cli@1.0.4
+        with:
+          inlineScript: |
+            az containerapp env create \
+            --name cae-${{ env.product }}-${{ env.target }}-001 \
+            --infrastructure-subnet-resource-id $(az network vnet subnet show --name snet-${{ env.product }}-001 --vnet-name vnet-${{ env.product }}-001 --query 'id' -o tsv) \
+            --tags ${{ env.tags }}
+
+      - name: Container app - MDM 📄
+        uses: Azure/cli@1.0.4
+        with:
+          inlineScript: |
+            az containerapp create \
+            --name ca-${{ env.product }}-mdm-${{ env.target }}-001 \
+            --environment cae-${{ env.product }}-${{ env.target }}-001 \
+            --registry-server cr${{ env.product }}${{ env.target }}001.azurecr.io \
+            --registry-username $(az acr credential show -n cr${{ env.product }}${{ env.target }}001 --query username | tr -d '"') \
+            --cpu 1.0 \
+            --memory 2.0Gi \
+            --min-replicas 1 \
+            --max-replicas 4 \
+            --ingress external \
+            --target-port ${{ vars.MDM_PORT }} \
+            --revisions-mode multiple \
+            --transport auto \
+            --tags ${{ env.tags }}
+
+      - name: Container app - TFS 📄
+        uses: Azure/cli@1.0.4
+        with:
+          inlineScript: |
+            az containerapp create \
+            --name ca-${{ env.product }}-tfs-${{ env.target }}-001 \
+            --environment cae-${{ env.product }}-${{ env.target }}-001 \
+            --registry-server cr${{ env.product }}${{ env.target }}001.azurecr.io \
+            --registry-username $(az acr credential show -n cr${{ env.product }}${{ env.target }}001 --query username | tr -d '"') \
+            --cpu 1.0 \
+            --memory 2.0Gi \
+            --min-replicas 1 \
+            --max-replicas 4 \
+            --ingress external \
+            --target-port ${{ vars.MDM_PORT }} \
+            --revisions-mode multiple \
+            --transport auto \
+            --tags ${{ env.tags }}
+
+      - name: Container app - ESTORE 📄
+        uses: Azure/cli@1.0.4
+        with:
+          inlineScript: |
+            az containerapp create \
+            --name ca-${{ env.product }}-estore-${{ env.target }}-001 \
+            --environment cae-${{ env.product }}-${{ env.target }}-001 \
+            --registry-server cr${{ env.product }}${{ env.target }}001.azurecr.io \
+            --registry-username $(az acr credential show -n cr${{ env.product }}${{ env.target }}001 --query username | tr -d '"') \
+            --cpu 1.0 \
+            --memory 2.0Gi \
+            --min-replicas 1 \
+            --max-replicas 4 \
+            --ingress external \
+            --target-port ${{ vars.MDM_PORT }} \
+            --revisions-mode multiple \
+            --transport auto \
+            --tags ${{ env.tags }}              
```

Comment:
I assume you're already aware, but just in case you forget, I think you'll want to delete these at some point

Reply:
Can I check why we need to delete them?

Reply:
I assumed we'd want the infrastructure for ESTORE and TFS to be controlled by actions in their own repos, rather than in MDM's repo

Reply:
I have removed respective `TFS` and `ESTORE` IaC commands since they will be handled in their respective repo, they were added purely for testing purpose.